### DISCUSSION TITLE
mobile app imcompatible

### DISCUSSION THREAD

**Discussion by vincentDcmps** [immich-app/immich#9387] [Q&A] [answered]
Hi try to install immich on an old phone Ith Android 7 but he is not compatible do we know why?

**Comment by mmomjian** [comment, marked as answer]
Google requires us to target a newer android version to be approved on the play store. I think someone was looking to compile an older version themself but I don’t know if that ever happened.

**Reply by NicholasFlamy** [reply]
I haven't quite gotten to doing that.

**Comment by NicholasFlamy** [comment]
This is a duplicate of #8917 but I'm not sure if that's also a duplicate.
